tests/tf-manifests/aws/account-roles/variables.tf

```
@@ -5,7 +5,7 @@ variable rhcs_environment {
 
 variable openshift_version {
     type = string
-    default = "4.13"
+    default = "4.13.1"
```

account role should be major-v.minor_v
```suggestion
    default = "4.13"
```